example/astx/ast.bnf

```diff
@@ -12,13 +12,13 @@ id : (_letter | '_') {_idchar} ;
 
 /* Syntax elements */
 
-<< import "github.com/goccmack/gocc/example/astx/ast" >>
+<< import "github.com/maxcalandrelli/gocc/example/astx/ast" >>
```

Comment:
This needs to say goccmack

Reply:
@maxcalandrelli you can make this work locally by doing the following:

```
$ cd $GOPATH/github.com/maxcalandrelli/gocc
$ go mod tidy
$ go mod edit --replace github.com/goccmack/gocc=./
```

just remember not to commit the replace in your go.mod